core/src/components/chip/chip.ios.vars.scss

```diff
@@ -0,0 +1,7 @@
+@import "../../themes/ionic.globals.ios";
+
+// iOS Chip
+// --------------------------------------------------
+
+/// @prop - Font size of the chip
+$chip-ios-font-size: clamp(13px, 0.875rem, 22px);
```

Comment:
I wasn't sure where to get the min & max values from. I did some quick searching and couldn't find any iOS spec for chip font sizes.